entsoe/parsers.py

```diff
@@ -371,19 +371,23 @@ def _available_period(timeseries: bs4.BeautifulSoup) -> list:
 
 
 def _unavailability_timeseries(soup: bs4.BeautifulSoup) -> list:
-    #if not ts:
-    #    return
+
+    # Avoid attribute errors when some of the fields are void: 
+    get_attr = lambda attr: "" if soup.find(attr) is None else soup.find(attr).text 
+    # When no nominal power is given, give default numeric value of 0:
+    get_float = lambda val: 0 if val == "" else float(val)
```

Comment:
Wouldn't it be more prudent to return `float('NaN')` if no value is given, instead of 0?

```suggestion
    get_float = lambda val: float('NaN') if val == "" else float(val)
```